pkg/asset/agent/image/ignition.go

```diff
@@ -360,6 +373,53 @@ func addExtraManifests(config *igntypes.Config, extraManifests *manifests.ExtraM
 	}
 }
 
+func getOSImagesInfo() (*models.OsImage, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+	defer cancel()
+
+	st, err := rhcos.FetchCoreOSBuild(ctx)
```

Comment:
fwiw I think it's fine to use `context.Background()` here. It should be getting the stream info from the file built in to the binary. (TBH I don't understand why this requires passing a context at all.)

Reply:
Ditto. There isn't any need to create the `ctx` instead use `context.Background()` directly. `pkg.rhcos.builds.FetchCoreOSBuild()` calls `FetchRawCoreOSStream()` which even then does not use the `ctx` object. I will create a seperate PR to remove the parameter but till then we can just pass `nil` or `context.Background()`